### ISSUE TITLE
Require this package when minimum-stability is "stable" ?

### ISSUE THREAD

**Issue by LittleBigDev** [portphp/portphp#48]
Hello,

Because of "stable" minimum-stability, I'm not able to require portphp/portphp. 
```
[InvalidArgumentException]                                                                                          
  Could not find package portphp/portphp at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability 
```

I tried changing to "dev" and require works fine. But for our project, "dev" minimum-stability is not allowed in production environment.

Do you know a workaround to require a stable version of portphp/portphp ?

Thanks by advance,
Bastien.

**Comment by aistis-** [comment]
Hi, how about in `composer.json`:

```
{
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

**Comment by LittleBigDev** [comment]
Hi @aistis- , 
I am not allowed to change the minimum stability because it's part of a bigger project.
I was able to require it though, by manually writing in composer.json and running composer install. No stability warning (which is weird IMO).

**Comment by discordier** [comment]
Simply put: `"portphp/portphp": "^1.0@dev"` instead of `"portphp/portphp": "^1.0"` in your `composer.json` .
This will change the `minimum-stability` for this package only.

**Comment by LittleBigDev** [comment]
Thanks I'll try that (not at work right now)

**Comment by ddeboer** [comment]
I just released portphp/portphp 1.0.0.
